### PR TITLE
Load data from file at start of program

### LINKED ISSUE #22: Implement loading data from file

### PR DESCRIPTION
**Pull request by kerct**
Fixes #22

### CODE REVIEW

File: src/main/java/seedu/modtracker/ModTracker.java [hunk 1]
```diff
@@ -36,11 +45,24 @@ public String start() {
...
+    private void loadData() {
+        try {
+            Parser parser = new Parser();
+            Scanner reader = storage.getReader();
+            while (reader.hasNext()) {
+                String input = reader.nextLine();
+                parser.parse(input, modList, "", storage, false);
+            }
+        } catch (FileNotFoundException e) {
+            ui.printErrorMessage(e.getMessage());
```
Comment (limgl1998) [inline src/main/java/seedu/modtracker/ModTracker.java:65]: Maybe can include a file creation here so the user need not manually create one

File: src/main/java/seedu/modtracker/ModTracker.java [hunk 2]
```diff
@@ -1,23 +1,32 @@
...
-    private ModuleList modList = new ModuleList();
 
     public static void main(String[] args) {
         new ModTracker().run();
     }
 
+    public ModTracker() {
+        ui = new Ui();
+        modList = new ModuleList();
+        storage = new Storage("data/modtracker.txt");
```
Comment (limgl1998) [inline src/main/java/seedu/modtracker/ModTracker.java]: I think the address can be refactored into a constant e.g. FILE_PATH

File: src/main/java/seedu/modtracker/ModTracker.java (hunk 1 above)
Comment (kerct) [reply src/main/java/seedu/modtracker/ModTracker.java:65]: The file is already created when the Storage object is initialised, this part is just a check when trying to read from the Storage object.

File: src/main/java/seedu/modtracker/ModTracker.java (hunk 2 above)
Comment (kerct) [reply src/main/java/seedu/modtracker/ModTracker.java]: Okay!

**Pull request merged**